src/lib/constants.ts

```diff
@@ -145,7 +143,7 @@ export const VIEW_NAMES_SEMANTIC = {
 export const SIDEBAR_BUTTON_SUFFIX = '-pane-button'
 
 /** Custom URL protocol our desktop registers */
-export const ZOO_STUDIO_PROTOCOL = 'zoo-studio:'
```

Comment:
This was one of the biggest blockers to working deep links.

Reply:
Crazy, great find, I don't know why this was working for me on Mac before your fixes.